### PR TITLE
Clean up unused functions and consts

### PR DESCRIPTION
**Pull request by feiskyer**
**What this PR does / why we need it**:

Clean up unused functions and consts.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cloudprovider/providers/azure/azure_controllerCommon.go [hunk 1]
```diff
@@ -63,8 +61,6 @@ type controllerCommon struct {
 	managementEndpoint    string
 	tokenEndPoint         string
 	aadResourceEndPoint   string
```
Comment (karataliu) [inline pkg/cloudprovider/providers/azure/azure_controllerCommon.go]: clientID, clientSecret, managementEndpoint, tokenEndPoint, aadResourceEndPoint are also not in use, consider removing them all.

File: pkg/cloudprovider/providers/azure/azure_test.go [hunk 2]
```diff
@@ -862,6 +862,9 @@ func getTestCloud() (az *Cloud) {
 		},
 	}
 	az.LoadBalancerClient = newFakeAzureLBClient()
+	az.RoutesClient = newFakeRoutesClient()
+	az.DisksClient = newFakeDisksClient()
+	az.RouteTablesClient = newFakeRouteTablesClient()
```
Comment (karataliu) [inline pkg/cloudprovider/providers/azure/azure_test.go]: Better keep this list sorted?

**Comment by feiskyer** [comment]
@karataliu Addressed comments. PTAL

**Comment by karataliu** [comment]
Looks that `az.LoadBalancerClient = ` still out of order?

**Comment by feiskyer** [comment]
oops, fixed.

**Pull request merged**